apps/patient/src/views/Status.tsx

```diff
@@ -15,6 +26,67 @@ import { formatAddress, getFulfillmentType, preparePharmacy } from '../utils/gen
 import { orderStateMapping as m, text as t } from '../utils/text';
 import { useOrderContext } from './Main';
 import { HorizontalStatusStepper } from '../components/HorizontalStatusStepper';
+import dayjs from 'dayjs';
+
+function formatReadyTime(baseTime: string, minutesUntilReady: number): ReactElement {
```

Comment:
The naming here really is bad. This returns the **confirmed** ready time. But we have a different field called `readyByTime` which is the **desired** pickup time.

We should fix the naming on both ends probs, but in the meantime this should probably be `<ReadyForPickupTime>` or something like that.